lib/vcr/cassette/erb_renderer.rb

```diff
@@ -40,8 +40,9 @@ def template
       end
 
       @@struct_cache = Hash.new do |hash, attributes|
-        attributes.map!(&:to_sym)
-        hash[attributes] = Struct.new(*attributes)
+        attributes = attributes.map(&:to_sym)
+        hash[attributes] = Struct.new(*attributes) unless hash.key?(attributes)
+        hash[attributes]
```

Comment:
@krainboltgreene Your suggested change got me thinking about reducing the number of keys in the cache. All of the following end up creating Structs with the same members and should reuse the cached Struct rather than create a new one:

```ruby
@@struct_cache[ [:a, :b] ]
@@struct_cache[ [:a, 'b'] ]
@@struct_cache[ ['a', :b] ]
@@struct_cache[ ['a', 'b'] ]
```